### PR TITLE
Add more credentials documentation

### PR DESCRIPTION
**Pull request by thesquelched**
Add more credentials documentation and examples.

### CODE REVIEW

File: docs/api/credentials.rst [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+    >>> lava.clusters.create(
+    ...     'my_cluster',
+    ...     'HADOOP_HDP2_2',
+    ...     username='scott',
+    ...     ssh_keys=[ssh_key.id],
+    ...     node_groups={'slave': {'count': 3, 'flavor_id': 'hadoop1-7'}})
+
+Similarly, you can add credentials to access other services from your clusters,
+e.g. `Cloud Files <http://www.rackspace.com/cloud/files>`_, which you can
+enable using the `connectors` argument when creating a cluster::
```
Comment (PaulMontgomery) [inline docs/api/credentials.rst]: Perhaps we should document the full list of connectors or indicate that cloud files is all that is available for now?

**Comment by PaulMontgomery** [comment]
+1

File: docs/api/credentials.rst (hunk 1 above)
Comment (thesquelched) [inline docs/api/credentials.rst]: I'll make a separate PR that adds a method for the `/credentials/types` method, as well as some docs.

Comment (thesquelched) [inline docs/api/credentials.rst]: On second though, we have methods for each of the credentials types, so it'd be better just to document it.

Comment (thesquelched) [inline docs/api/credentials.rst]: Which I've done.

**Comment by rnirmal** [comment]
+1

**Pull request merged**